docs/src/add-services/mysql/_index.md

```diff
@@ -348,6 +348,31 @@ ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_
 
 For further details, see the [MariaDB documentation](https://mariadb.com/kb/en/character-set-and-collation-overview/).
 
+## Password generation
+ 
+When you create an endpoint to allow your app and database to communicate,
+an empty password is generated by default.
```

Comment:
:question: I thought that the empty password was when you didn't specify an endpoint. Is that not true? It was my impression from the Context that you could generate a password specifically by defining an endpoint.